source/operator.rst

```diff
@@ -2743,6 +2743,7 @@ Percona Server for MongoDB backups.
 +-----------------+---------------------------------------------------------------------------------------------+
 | **Example**     | ``gzip``											|
 +-----------------+---------------------------------------------------------------------------------------------+
-| **Description** | The backup compression format								|
```

Comment:
Actually, Zstandard is also supported since https://github.com/percona/percona-backup-mongodb/commit/40897fd6edc884e0b64a6d2d44891c27be4f467a.
And `s2` is supported as well (extension of Snappy).